concordium-sdk/src/main/java/com/concordium/sdk/responses/DelegatorInfo.java

```diff
@@ -0,0 +1,21 @@
+package com.concordium.sdk.responses;
+
+import com.concordium.sdk.responses.accountinfo.PendingChange;
+import com.concordium.sdk.transactions.AccountAddress;
+import com.concordium.sdk.transactions.CCDAmount;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.Optional;
+
+@Builder
+@Getter
+@EqualsAndHashCode
+@ToString
+public class DelegatorInfo {
+    private final AccountAddress account;
+    private final CCDAmount stake;
+    private final Optional<PendingChange> pendingChange;
```

Comment:
Could you please document the fields here. I assume documentation then gets propagated to the gettters.

Reply:
done